packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -1380,27 +1213,21 @@ class _InkResponseState extends State<_InkResponseStateWidget>
 /// This is because ink splashes draw on the underlying [Material] itself, as
 /// if the ink was spreading inside the material.
 ///
-/// The [Ink] widget can be used as a replacement for [Image], [Container], or
-/// [DecoratedBox] to ensure that the image or decoration also paints in the
-/// [Material] itself, below the ink.
+/// Replacing the opaque widget with a [Material] allows the ink splash to be shown.
 ///
-/// If this is not possible for some reason, e.g. because you are using an
-/// opaque [CustomPaint] widget, alternatively consider using a second
-/// [Material] above the opaque widget but below the [InkWell] (as an
-/// ancestor to the ink well). The [MaterialType.transparency] material
-/// kind can be used for this purpose.
+/// Alternatively, an [InkBox] can be set as the opaque widget's child, and
+/// ink effects will be visible on top of it.
 ///
 /// ### InkWell isn't clipping properly
 ///
-/// If you want to clip an InkWell or any [Ink] widgets you need to keep in mind
-/// that the [Material] that the Ink will be printed on is responsible for clipping.
```

Comment:
A nice side effect of rewording this documentation is that it follows the [style guide](https://github.com/flutter/flutter/blob/80a65bd781bc81fbbeba00430646ae8f9f6832c2/docs/contributing/Style-guide-for-Flutter-repo.md#use-the-passive-voice-recommend-do-not-require-never-say-things-are-simple) now!